spec/controllers/admin/categories_controller_spec.rb

```diff
@@ -34,6 +34,14 @@
     end
   end
 
+  describe "test_new" do
```

Comment:
Do you need to test any other scenarios? Like validation errors? Here or in the cucumber scenarios?